app.js

```diff
@@ -80,9 +80,12 @@ app.use(function (req, res, next) {
 // store the client secret in regardless) it makes sense
 var errors = require("./lib/errors.js").ERRORS;
 app.use(function (req, res, next) {
-    // don't authenticate OPTIONS requests for browser compatbility
+    // don't authenticate OPTIONS requests for browser compatibility
     if (req.method === "OPTIONS") return next();
 
+    // don't authenticate health checks
+    if (req.path.indexOf("/health") >= 0) return next();
```

Comment:
Can we use `===` rather than `indexOf` here? Given this is a health app, it's not too unlikely we'll want to add a nested `/health` endpoint (`/patients/:id/health` or something) in the future, and using `indexOf` here could cause silent issues with authentication.